src/AppBundle/Controller/SecurityController.php

```diff
@@ -13,6 +13,7 @@
 
```

Comment:
These SecurityController.php changes are the perfect example of why I don't like this new way of developing in Symfony. You remove 1 easy-to-understand line (`$helper = $this->get('security.authentication_utils');`) and replace it with 6 lines of code ... including some "class guessing" to import the `AuthenticationUtils` class 😕